### ISSUE TITLE
checklist of accessibility issues reported by Wave

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#940]
The following is a list of accessibility issues reported by wave when viewing each page of our page types demonstrator https://xot.xerte.org.uk/play.php?template_id=116#xertepagetypes
Some of these (but not all) are listed as separate issues all tagged under accessibility but it seemed worth adding a checklist that we can tick off individual items either as they are done or as the individual issues are done and closed. I've also edited the list and added my id to the one's I'm already working on so perhaps we can all do the same as we start working on individual issues?

- [x] audio slideshow empty headings show as errors #938 @FayCross 
- [ ] low contrast report on flashcard page backface visibility @JohnSmith-LT (MathJax - need to have scss rules added for theme)
- [x] hotspot image connector 2 missing alt text errors - image map area @JohnSmith-LT 
- [x] Annotated diagram 26x missing links reported all have href="#" @FayCross 
- [x] Answer x of y #correctAnswer H3 missing heading?@FayCross
- [x] Dialogue missing form labels #925 @JohnSmith-LT 
- [x] Dictation missing form labels answerTxt #925 @JohnSmith-LT 
- [x] Documentation missing form labels #925 @JohnSmith-LT 
- [x] Drag and drop labelling missing form labels on submit button #925 (also drop zone for 1 etc for screen reader users?)
- [x] Gap fill drag and drop missing form labels submit button and show button #925
- [x] Gap fill drop down missing form labels #925
- [x] Gap fill text entry missing form labels #925
- [x] Hotspot image missing alt text @JohnSmith-LT 
- [x] Interactive diagram missing alt text @FayCross 
- [x] Model answer results low contrast on red tip text @ronm123 
- [x] Modify exercise missing form labels #925
- [x] Opinion missing form labels #925
- [x] Sortable grid low contrast errors @ronm123
- [x] text correct missing form labels #925
- [x] Youtube feed missing alt text and low contrast errors @JohnSmith-LT 
- [x] Results page empty table header
- [x] Results page contrast errors
